lib/atlas/graph_from_yaml.rb

```diff
@@ -0,0 +1,63 @@
+module Atlas
+  class GraphFromYaml
+    NODE_ATTRIBUTES = %i(demand max_demand electricity_output_capacity cc_in cc_out)
```

Comment:
:typical_input_capacity is missing - but see below for a more general remark about this list